### DISCUSSION TITLE
ошибка входа в кошелек Iota Firefly

### DISCUSSION THREAD

**Discussion by IgorDev22** [iotaledger/firefly#7897] [Help & Support] [answered]
![ошибка](https://github.com/iotaledger/firefly/assets/128074097/d990a68c-63d3-415c-a7bd-1c954cde887f)
ошибка появляется после обновления программы, фото скрин прилагаю
![ошибка](https://github.com/iotaledger/firefly/assets/128074097/a57d99e4-2e71-4c42-9968-8bb5d0989c71)

**Comment by begonaalvarezd** [comment, marked as answer]
Your machine time is not correct, please update your system time by syncing it or setting it to automatic and try to login again, the issue will be gone when your machine clock is correct
